README.md

```diff
@@ -702,11 +705,80 @@ Name        | Value example | Description
 
 #### Service props
 
-Name             | Value example                                                         | Description
------------------|-----------------------------------------------------------------------|-------------
-`sup_components` | ["is_secured", "door_is_closed", "bolt_is_locked", "latch_is_closed"] | List of supported lock component components
-`min_volume`     | 0                                                                     | Minimum volume level
-`max_volume`     | 2                                                                     | Maximum volume level
+Name                        | Value example                                                         | Description
+----------------------------|-----------------------------------------------------------------------|-------------
+`sup_components`            | ["is_secured", "door_is_closed", "bolt_is_locked", "latch_is_closed"] | List of supported lock component components.
+`min_volume`                | 0                                                                     | Minimum volume level.
+`max_volume`                | 2                                                                     | Maximum volume level.
+`supports_auto_relock`      | true                                                                  | Defines if device supports auto-relock capability.
+`supports_hold_and_release` | false                                                                 | Defines if device supports hold and release capability.
+`supports_block-to-block`   | true                                                                  | Defines if device supports block-to-block capability.
+`supports_twist_assist`     | false                                                                 | Defines if device supports twist assist capability.
+
+#### Example
+
+```json
+{
+  "serv": "door_lock",
+  "type": "cmd.lock.set_configuration",
+  "val_t": "object",
+  "val": {
+    "operation_type": "timed",
+    "outside_handles_mode": {
+      "handle1": 1,
+      "handle2": 1,
+      "handle3": 1,
+      "handle4": 1,
+    },
+    "inside_handles_mode": {
+      "handle1": 1,
+      "handle2": 1,
+      "handle3": 1,
+      "handle4": 1,
+    },
+    "lock_timeout_minutes": 2,
+    "lock_timeout_seconds": 1,
+    "auto_relock_time": 60,
+    "hold_and_release_time": 55,
+    "block_to_block": true,
+    "twist_assist": false
+  },
+  "ver": "1",
+  "props": null,
+  "tags": null
+}
+
+```
+
+#### Notes
+
+Messages cmd.lock.set_configuration and evt.lock.configuration_report will look exactly the same according to value body.
+
+Explanation of cmd.lock.set_configuration interface value properties.
+
+- operation_type - One of sup_operation_types property within service. At the moment supported values are: 'timed' and 'constant'.
```

Comment:
within `the` service

but more importantly, these values (`timed` and `constant`) should preferably be mentioned somewhere in props

Reply:
done

Reply:
but if that's not mentioned in props, then the app has no idea what it should show, unless we will start hardcoding values according to protocol type, which is bad

Reply:
Ohh, You mean, create service property e.g. "supported_op_types"?

Reply:
yes, if it's adapter dependent, then there should be a clear way through which the app would be able to get that info (and not hardcode it for zwave adapter)